integration/helpers/vite-deno-template/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "integration-vite-deno-template",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "build": "deno run -A npm:@remix-run/dev vite:build",
+    "dev": "deno run -A npm:@remix-run/dev vite:dev",
+    "typecheck": "deno check '**/*' && deno run -A npm:typescript/tsc",
+    "typegen": "deno types > ./app/deno.d.ts"
+  },
+  "dependencies": {
+    "@remix-run/dev": "workspace:*",
+    "@remix-run/node": "workspace:*",
+    "@remix-run/react": "workspace:*",
+    "@remix-run/serve": "workspace:*",
+    "@remix-run/server-runtime": "workspace:*",
+    "isbot": "^4.1.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@remix-run/dev": "workspace:*",
+    "@types/react": "^18.2.20",
+    "@types/react-dom": "^18.2.7",
+    "typescript": "^5.1.0",
+    "vite": "5.1.3",
+    "vite-tsconfig-paths": "^4.2.1"
+  },
+  "dependenciesMeta": {
+    "@remix-run/dev": {
+      "injected": true
+    },
+    "@remix-run/node": {
+      "injected": true
+    },
+    "@remix-run/react": {
+      "injected": true
+    },
+    "@remix-run/serve": {
+      "injected": true
+    },
+    "@remix-run/server-runtime": {
+      "injected": true
+    }
+  }
```

Comment:
this is necessary for deno as these packages are cjs and not esm. deno will refuse to run cjs code that's inside our workspace which is what happens due to the node_modules packages being symlinked by default with pnpm. [`dependenciesMeta.injected`](https://pnpm.io/package_json#dependenciesmetainjected) tells pnpm to install these workspace dependencies without using symlinks which convinces deno that these are npm packages (as they're now inside the `node_modules` folder and not symlinked out of there) and enables cjs for it.